api-docs/swagger-v1.1.yml

```diff
@@ -1120,6 +1143,12 @@ parameters:
     minimum: 1
     maximum: 10
     required: false
+  coded:
+    name: coded
+    type: boolean
+    in: query
+    description: restrict to nutrients having a code
+    required: false
```

Comment:
Eigenlijk wel, maar dat is hetzelfde als weglaten. Dan houd ik liever de standaard op of aanwezig met `true` of afwezig. Andere varianten zijn toegift.